src/dense-arrays.lisp

```diff
@@ -75,8 +75,7 @@
                      (layout *array-layout*)
 
                      (displaced-to nil displaced-to-p)
-                     (offsets nil offsets-p)
-                     (displaced-index-offset 0 displaced-index-offset-p))
+                     (offset 0))
```

Comment:
Since `dense-arrays:make-array` is supposed to be an almost drop-in replacement for `cl:make-array`, could you think rename the `offset` argument to `displaced-index-offset`?